textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/DoubleField.java

```diff
@@ -0,0 +1,47 @@
+package edu.uci.ics.textdb.common.field;
+
+import edu.uci.ics.textdb.api.common.IField;
+
+public class DoubleField implements IField{
+    
+    private Double value;

```

Comment:
With the current design of IField interface, we cannot do it. Shall I remove getValue() method from IField interface?